packages/esm-ward-app/src/types/index.ts

```diff
@@ -9,18 +10,39 @@ import type {
 } from '@openmrs/esm-framework';
 import type React from 'react';
 
-export type WardPatientCardRow = React.FC<WardPatient>;
-export type WardPatientCardElement = React.FC<WardPatient>;
+export type WardPatientCard = React.FC<WardPatient>;
 
 // WardPatient is a patient admitted to a ward, and/or in a bed on a ward
 export type WardPatient = {
+  /**
+   * The patient and their current visit. These values are taken either
+   * from either the inpatientAdmission object, the inpatientRequest object
+   * or the admissionLocation object (which contains the bed)
+   */
   patient: Patient;
   visit: Visit;
+
+  /**
+   * the bed assigned to the patient. This object is only set if the patient
+   * has a bed assigned
+   */
   bed?: Bed;
-  admitted: boolean;
-  encounterAssigningToCurrentInpatientLocation: Encounter;
-  firstAdmissionOrTransferEncounter: Encounter;
+
+  /**
+   * The admission detail. This object is only set if the patient has been
+   * admitted to the ward
+   */
+  inpatientAdmission: InpatientAdmission;
```

Comment:
Doesn't this need a question mark? Or are you enforcing that it is explicitly set to null?

Reply:
Yeah, I'd rather explicitly set these to null. (In fact, let me remove the `?` from the `bed` field)